src/main/kotlin/com/colivery/serviceaping/persistence/entity/UserEntity.kt

```diff
@@ -43,8 +43,16 @@ data class UserEntity(
         var phone: String,
 
         @Column(nullable = false)
-        val firebaseUid: String
+        val firebaseUid: String,
+
+        @Enumerated(EnumType.STRING)
+        @Column(nullable = false)
+        var role: Role = Role.USER
 ) {
+    enum class Role {
+        USER, HOTLINE, ADMIN
```

Comment:
What exactly stands the role "HOTLINE" for?

Reply:
we have a ticket (https://colivery.atlassian.net/browse/API-17) specifying the 3 roles.. but if we don't need it, we can remove it